plugins/router/f5/f5.go

```diff
@@ -307,15 +318,24 @@ func newF5LTM(cfg f5LTMCfg) (*f5LTM, error) {
 		privkeyFileName = newPrivkeyFile.Name()
 	}
 
+	partitionPath := F5DefaultPartitionPath
+	if len(cfg.partitionPath) > 0 {
+		partitionPath = cfg.partitionPath
+	}
```

Comment:
Fair enough regarding your first point, though I thought `Complete()` would handle the case where the partition path were set to an empty string.